Docs/AcpiSamples/SSDT-PLUG.dsl

```diff
@@ -20,24 +20,26 @@ DefinitionBlock ("", "SSDT", 2, "ACDT", "CpuPlug", 0x00003000)
     External (_SB_.SCK0.CP00, ProcessorObj)
     External (_SB_.SCK0.PR00, ProcessorObj)
 
-    Method (PMPM, 4, NotSerialized) {
-       If (LEqual (Arg2, Zero)) {
-           Return (Buffer (One) { 0x03 })
-       }
-
-       Return (Package (0x02)
-       {
-           "plugin-type", 
-           One
-       })
-    }
-
     If (CondRefOf (\_SB.CPU0)) {
         If ((ObjectType (\_SB.CPU0) == 0x0C)) {
             Scope (\_SB.CPU0) {
-                Method (_DSM, 4, NotSerialized)  
-                {
-                    Return (PMPM (Arg0, Arg1, Arg2, Arg3))
+                If (_OSI ("Darwin")) {
```

Comment:
Is it any harmful? I am actually unsure there are any examples of these methods inducing harm in other systems.

Reply:
starting windows from opencore, you noticed or temp high cpu and fans turn fast, with darwin method ssdt plug, then start windows from oc and temp low and fans low regimes

Reply:
weird, but ok.